### DISCUSSION TITLE
BT-1252. Optional or mandatory field

### DISCUSSION THREAD

**Discussion by fernarr** [OP-TED/eForms-SDK#354] [Q&A] [answered]
Hello,

The BT-1252 field, Direct Award Justification Previous Procedure Identifier, is used to indicate the UUID to the previous procedure in case of a Direct Award.
However, we would like to know if there are some cases in which this field should be required or if it is an optional field, in business rules.

Sincerely

**Comment by YvesJo** [comment, marked as answer]
Hi,
There are some rules that enforce the existence of BT-1252:

- If Direct Award Justification Code (BT-136) value is equal to "No suitable tenders, requests to participate, or applications were received in response to a previous notice", then Direct Award Justification Previous (BT-1252) must be present.	(BR-0053-0001)
- If Direct Award Justification Code (BT-136) value is equal to "Service contract to be awarded to the winner(s) under the rules of a design contest", then Direct Award Justification Previous (BT-1252) must be present.	(BR-0053-0002)
- If Direct Award Justification Code (BT-136) value is equal to "Partial replacement or extension of existing supplies or installations by the original supplier ordered under the strict conditions stated in the Directive", then Direct Award Justification Previous (BT-1252) must be present.	(BR-0053-0003)
- If Direct Award Justification Code (BT-136) value is equal to "New works or services, constituting a repetition of existing works or services and ordered in accordance with the strict conditions stated in the Directive", then Direct Award Justification Previous (BT-1252) must be present.	(BR-0053-0004)

For such non-technical questions, please get in touch directly with the helpdesk: ted@publications.europa.eu
I also mark this discussion as answered.
KR,

**Reply by fernarr** [reply]
Thank you for the answer,

however, I can't find these rules in the [eforms-SDK](https://github.com/OP-TED/eForms-SDK/) or in the [documentation](https://docs.ted.europa.eu/eforms/latest/index.html).  

I have founf in the [schematrons files](https://github.com/OP-TED/eForms-SDK/tree/develop/schematrons) and I am not be able to find these rules.  In the file [rule_en.xml](https://raw.githubusercontent.com/OP-TED/eForms-SDK/develop/translations/rule_en.xml) I am fin the rules BR-BT-01252-00** with these restrictions, but I think that it is neccesaty that the rules exist in schematrons files.

Could you tell me if these specific rules exist in some files, document or list?

Thank you for all the information.

**Reply by YvesJo** [reply]
These rules are currently deactivated since SDK 1.3 and will get reactivate with the time. Once active, they will appear again in the schematron files.
You may however find them in SDK 1.2.
